### PR TITLE
Iterate on layout.

### PR DESCRIPTION
**Pull request by anthonyshew**
## 🎯 Changes

Trying out a different layout for examples. Perhaps a bit preferential but I think this layout presents the information with less noise and better clarity. It also gives us a way to show the package manager tabs for the examples that have them.

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/next/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/docs/main/example-apps.mdx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+- Example deploy: [nextjs.trpc.io](https://nextjs.trpc.io)
+- [Source code](https://github.com/trpc/examples-next-prisma-starter)
+
+---
+
+### zART-stack example (zero-API, React, TypeScript)
+
+<Tabs>
+  <TabItem value="git clone" label="git clone">
```
Comment (juliusmarminge) [inline www/docs/main/example-apps.mdx:51]:
```suggestion
  <TabItem value="git" label="git">
```

File: www/docs/main/example-apps.mdx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+  </TabItem>
+</Tabs>
+
+- Example deploy: [nextjs.trpc.io](https://nextjs.trpc.io)
+- [Source code](https://github.com/trpc/examples-next-prisma-starter)
+
+---
+
+### zART-stack example (zero-API, React, TypeScript)
```
Comment (juliusmarminge) [inline www/docs/main/example-apps.mdx:48]: This project is deprecated and won't be updated to v10. Maybe we should link https://github.com/t3-oss/create-t3-turbo instead?

CleanShot 2022-11-17 at 12 48 05@2x

File: www/docs/main/example-apps.mdx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+- Example deploy: [todomvc.trpc.io](https://todomvc.trpc.io)
+- [CodeSandbox](https://codesandbox.io/s/github/trpc/trpc/tree/main/examples/next-prisma-todomvc?file=/pages/%5Bfilter%5D.tsx)
+- [Source code](https://github.com/trpc/examples-next-prisma-todomvc)
+
+---
+
+### Express server with tRPC Shield and Prisma
+
+<Tabs>
```
Comment (juliusmarminge) [inline www/docs/main/example-apps.mdx:131]: Any reason you didn't include yarn/pnpm tabs here?

**Review by juliusmarminge — COMMENTED** [review]
I like this layout 👍🏼

File: www/docs/main/example-apps.mdx (hunk 3 above)
Comment (anthonyshew) [reply www/docs/main/example-apps.mdx:131]: I wasn't sure if `create-trpc-appx` was going to work with `yarn` and `pnpm` so I left it to just npx. I can add if they will!

**Comment by anthonyshew** [comment]
The other couple things on here are stuff that I don't really have great context on so I will leave those to you folks!

**Review by sachinraja — APPROVED** [review]
I like it!

File: www/docs/main/example-apps.mdx (hunk 2 above)
Comment (KATT) [reply www/docs/main/example-apps.mdx:48]: Yes! feel free to edit @juliusmarminge

**Review by KATT — APPROVED** [review]

**Review by KATT — APPROVED** [review]

**Review by KATT — APPROVED** [review]

File: www/docs/main/example-apps.mdx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+---
+
+### zART-stack example (zero-API, React, TypeScript)
+
+<Tabs>
+  <TabItem value="git" label="git">
+
+```bash
+git clone git@github.com:KATT/zart.git
```
Comment (juliusmarminge) [inline www/docs/main/example-apps.mdx:54]:
```suggestion
git clone git@github.com:t3-oss/create-t3-turbo.git
```

File: www/docs/main/example-apps.mdx [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+  </TabItem>
+</Tabs>
+
+- [Source code](https://github.com/KATT/zART)
```
Comment (juliusmarminge) [inline www/docs/main/example-apps.mdx:60]:
```suggestion
- [Source code](https://github.com/t3-oss/create-t3-turbo)
```

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**